### DISCUSSION TITLE
Add `copy_message` method

### DISCUSSION THREAD

**Discussion by MoSi8160** [kianahmadian/python-bale-bot#57] [Q&A] [answered]
how can i copy message from chat A to chat B? (its just forward but without quoting)

**Comment by kian-ahmadian** [comment, marked as answer]
سلام عزیز وقت بخیر
این متد هنوز از طرف بله اضافه نشده. به محض اضافه شدن در ورژن های بعدی از طرف ما هم اضافه میشه.
